src/stylus/settings/_variables.styl

```diff
@@ -36,10 +36,10 @@ $material-twelve-percent-dark := rgba(#000, 0.12)
 // ============================================================
 $grid-breakpoints := {
   xs: 0
-  sm: 576px
-  md: 768px
-  lg: 992px
-  xl: 1200px
+  sm: 480px
+  md: 1024px
+  lg: 1440px
+  xl: 1920px
```

Comment:
According to the breakpoint table, small should start at 600.

Also the way I interpret their second note ("Desktop breakpoints are 16dp below the listed values to accommodate variations in browser chrome") is that values >= 1024 should be -16px, so 1008, 1264, etc